src/assets/scripts/InputController.js

```diff
@@ -467,17 +481,17 @@ export default class InputController {
 
             case KEY_CODES.LEFT_ARROW:
                 // shortKeys in use
-                if (window.gameController.game.option.get('controlMethod') === 'arrows') {
-                    this.$commandInput.val(`${currentCommandInputValue} \u2BA2`);
+                if (this._isArrowControlMethod()) {
+                    this.$commandInput.val(`${currentCommandInputValue}t l `);
```

Comment:
>I just looked through it and found where the callsign is added to the command line and removed the space there.

As long as you can still click to select an aircraft and type `[C] [A] [F] [enter]` without needing to add a leading space, that'll do, but it sounds like you're making it so selecting the aircraft inserts `AAL123` instead of `AAL123[space]` in the `currentCommandInputValue`.